CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Full documentation for hipCUB is available at [https://rocm.docs.amd.com/projects/hipCUB/en/latest/](https://rocm.docs.amd.com/projects/hipCUB/en/latest/).
 
+## hipCUB-x.x.x for ROCm 6.x.x
```

Comment:
```suggestion
## (Unreleased) hipCUB-x.x.x for ROCm 6.x.x
```
This ensures that it's not picked up by the release notes script until it's ready fior release.